src/client/pydaos/pydaos_shim.c

```diff
@@ -106,7 +106,7 @@ __shim_handle__daos_init(PyObject *self, PyObject *args)
 
 	rc = daos_init();
 	if ((rc == 0) && (use_glob_eq == 0)) {
-		override = getenv("PYDAOS_GLOB_EQ");
+		d_agetenv_str(&override, "PYDAOS_GLOB_EQ");
```

Comment:
This would be a case for d_getenv_bool() but I can make this change in other pydaos work I'm doing.

Reply:
If I have to repush, I will fix this.
Otherwise, if you can do it in your PR, it will be perfect: thanks for your proposal :)
Otherwise, I will do it in a follow up PR.